lambdas/service/app.py

```diff
@@ -860,69 +904,68 @@ def start_manifest_generation():
                     status_code=301 if wait_time else 302)
 
 
-@app.route('/fetch/manifest/files', methods=['GET'], cors=True)
+@app.route('/fetch/manifest/files', methods=['GET'], cors=True, path_spec={
+    'parameters': [
+        params.query(
+            'filters',
+            schema.optional(schema.object(
+                file=schema.object(
+                    **{facet_name: schema.object(**{'is': schema.array(str)})
+                       for facet_name in Plugin.load().service_config().translation.keys()}
+                )
+            )),
+            description='Filters to be applied when generating the manifest',
+        ),
+        params.query(
+            'format',
+            schema.optional(schema.enum('compact', 'full', 'terra.bdbag', type_=str)),
+            description=manifest_description,
+        ),
+        params.query(
+            'token',
+            schema.optional({'type': 'string'}),
+            description='Reserved. Do not pass explicitly.',
+        ),
+    ],
+}, method_spec={
+    'description': unwrap('''
+        <p>Initiate and check status of a manifest generation job, returning a
+        200 response with simulated headers in the body.
+        <p>If the client receives a response body with the `Status` field set
+        to 301, the client should wait the number of seconds specified in
+        `Retry-After` and then request the URL given in the `Location` field.
+        The URL will point back at this endpoint so the client should expect a
+        response of the same shape. Note that the actual HTTP response is of
+        status 200, only the `Status` field of the body will be 301. The intent
+        is to emulate HTTP while bypassing the default client behavior which,
+        in most web browsers, is to ignore `Retry-After`. The response described
+        here is intended to be processed by client-side Javascript such that the
+        recommended delay in `Retry-After` can be handled in Javascript rather
+        than relying on the native implementation by the web browser.
+        <p>The client should request the URL given in the `Location` field. The
+        URL will point to a different service and the client should expect a
+        response containing the actual manifest. Currently the `Location` field
+        of the final response is a signed URL to an object in S3 but clients
+        should not depend on that.
+    '''),
+    'responses': {
+        '200': {
+            'description': 'The manifest generation has been started or is still ongoing',
+            'content': {
+                'application/json': {
+                    'schema': schema.object(
+                        Status=int,
+                        Location={'type': 'string', 'format': 'url'},
+                        **{'Retry-After': schema.optional(int)}
+                    )
+                }
+            }
```

Comment:
This looks to be the same as https://github.com/DataBiosphere/azul/pull/1435/files#diff-a3dec4f6b35a08e7cb29086cb8766c3aR871-R879

and should be extracted as a variable as well to reduce redundancy.

Reply:
Ah.. I see it's not. Maybe there's a way to reduce the redundancy still.

Reply:
Extracted redundant documentation to a function to allow for the slight difference in the instances where it is used.